svelte-kit-scss/src/lib/components/Profile/OrganizationProfile/OrganizationDetails/OrganizationMembers.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { it, expect } from 'vitest';
+import { render } from '@testing-library/svelte';
+import OrganizationMembers from './OrganizationMembers.svelte';
+import { MOCK_GITHUB_ORGANIZATION_MEMBERS } from '$lib/helpers/mocks/users';
+
+const members = MOCK_GITHUB_ORGANIZATION_MEMBERS;
+
+describe('OrganizationMembers', () => {
+  function organizationMembersTree() {
+    return render(OrganizationMembers, {
+      members,
+    });
+  }
+
+  beforeEach(() => {
+    organizationMembersTree();
+  });
+
+  let currentTestIndex = 0;
+  it.each(members)(`should render member ${currentTestIndex}`, ({ href, avatarUrl }) => {
```

Comment:
```suggestion
  it.each(members)(`should render member %#`, ({ href, avatarUrl }) => {
```

You can use `%#` to inject the index into the test name, see https://vitest.dev/api/#test-each